README.md

```diff
@@ -0,0 +1,116 @@
+# Microsoft Authentication Library (MSAL) for Python
+
```

Comment:
As with all our MSAL SDKs, we should add **Preview** to the title and this note about Preview: https://github.com/AzureAD/microsoft-authentication-library-for-js#important-note-about-the-msal-preview.

In addition, we should probably mention that we have added certain flows and others are yet to come.

Reply:
OK, I'm adding that preview keyword into the title and adding that preview paragraph.

For the sake of getting this README published quicker (because a short readme is still better than no readme at all), I would suggest we do not mention flows in this version (besides, neither of MSAL .Net or MSAL javascript mentions any flows in their readme). But I totally agree that we would revisit this in (near?) future to give a concise flows coverage.

Reply:
I believe those details belong in the wiki not in the readme. You can point to a wiki page in the readme which talks about supported flows.